tests/profiles-enabled/test_shipments.py

```diff
@@ -1251,6 +1280,19 @@ def test_permission_link(self):
         response = self.client.get(f'{shipment_url}?permission_link={valid_permission_id_with_exp}')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
+        # Assert that an anonymous user with valid permission cannot modified a shipment
+        response = self.client.patch(f'{shipment_url}?permission_link={valid_permission_id_with_exp}',
+                                     shipment_update_info, content_type=content_type)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+        # An anonymous user cannot access a shipment's permission links
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+        # An anonymous user cannot access the shipment-list with a valid permission link
+        response = self.client.get(f'{url_shipment_list}?permission_link={valid_permission_id_with_exp}')
```

Comment:
And from line 1292 to line 1294 is the test for an unauthenticated user on `shipments-list` which returns `403`.